app/api/task_sheet/[id].tsx

```diff
@@ -0,0 +1,46 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { TaskSheet } from "../interfaces/interfaces";
+
+// Mock data
+const taskSheets: TaskSheet[] = [
+    {
+        id: 1,
+        tasks: JSON.stringify([
+            { task_id: 1, task_name: 'Homework', completed: false, user_ids: '[1, 2]' },
+            { task_id: 2, task_name: 'Chores', completed: true, user_ids: '[1, 2]' },
+        ]),
+    },
+    {
+        id: 2,
+        tasks: JSON.stringify([
+            { task_id: 3, task_name: 'Shopping', completed: false, user_ids: '[1]' },
+            { task_id: 4, task_name: 'Cleaning', completed: true, user_ids: '[2]' },
+        ]),
+    },
+];
+
+// GET function to fetch tasks for a specific TaskSheet
+export async function GetTasks(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
@jovanpet can correct me if i'm wrong, but I thought we agreed that the endpoints would be simple functions (with a number or array parameter) that can be imported anywhere in the front end. Then behind the scenes in this file we would perform the necessary API calls to AWS.